lib/wibox/layout/ratio.lua

```diff
@@ -334,18 +334,20 @@ function ratio:ajust_ratio(index, before, itself, after)
 
     self:emit_signal("widget::layout_changed")
 end
+ratio.ajust_ratio = ratio.adjust_ratio -- For backwards compatibility TODO v5: Remove this
```

Comment:
```suggestion
--- Update all widgets to match a set of a ratio.
--
-- This method is kept for backwards compatibility, please use `:adjust_ratio` instead.
-- @see wibox.layout.ratio.adjust_ratio
-- @deprecated wibox.layout.ratio.ajust_ratio
-- @method ajust_ratio
-- @tparam number index The index of the widget to change
-- @tparam number index The index of the widget to change
-- @tparam number before The sum of the ratio before the widget
-- @tparam number before The sum of the ratio before the widget
-- @tparam number itself The ratio for "widget"
-- @tparam number itself The ratio for "widget"
-- @tparam number after The sum of the ratio after the widget
-- @tparam number after The sum of the ratio after the widget
function ratio:ajust_ratio(...)
    require('gears.debug').deprecate(
        "Use `:adjust_ratio` rather than `:ajust_ratio`",
        { deprecated_in = 5 }
    )
    return self:adjust_ratio(...)
end
```